tests/metrics/genai/prompts/test_v1.py

```diff
@@ -84,22 +94,38 @@ def test_evaluation_model_output():
         - Score 5: the answer correctly answer the question and not missing any major aspect
 
     Examples:
-        Input: This is an input
-        Provided output: This is an output
-        Provided ground_truth: This is an output
-        Score: 4
-        Justification: This is a justification
-
-        Input: This is an example input 2
-        Provided output: This is an example output 2
-        Provided ground_truth: This is an output
-        Score: 4
-        Justification: This is an example justification 2
-
-    And you'll need to submit your grading for the correctness of the output,
-    using the following in json format:
-    Score: [your score number for the correctness of the output]
-    Justification: [your step by step reasoning about the correctness of the output]
+        Input:
+        This is an input
+
+        Output:
+        This is an output
+
+        Additional information used by the model:
+        key:
+        ground_truth
```

Comment:
Can we remove the extra newline for key and make sure that we aren't inserting this newline in the actual template?